examples/02_decoding/plot_haxby_decoder.py

```diff
@@ -0,0 +1,83 @@
+"""
+Decoding with decoder metaestimator: face vs house recognition
+==============================================================
+
+Here is a simple example of decoding with the decoder metaestimator,
+reproducing a face vs house discrimination task on the study Haxby 2001.
+
+    * J.V. Haxby et al. "Distributed and Overlapping Representations of Faces
+      and Objects in Ventral Temporal Cortex", Science vol 293 (2001),
+      p 2425.-2430.
+
+"""
+######################################################################
+# Loading the data
+######################################################################
+
+# First, we load the Haxby dataset
+from nilearn.datasets import fetch_haxby
+data_files = fetch_haxby()
+
+func_filenames = data_files.func[0]
+labels_filenames = data_files.session_target[0]
+
+# Load Target labels
+import numpy as np
+labels = np.recfromcsv(labels_filenames, delimiter=" ")
+
+# In this example, we restrict to face and house conditions
+target = labels['labels']
+condition_mask = np.logical_or(target == b"face", target == b"house")
```

Comment:
I agree it confusing. But, ATM we will keep ```b``` otherwise it would fail in Python 3. This is a separate issue addressing in a pending PR #1411.